inventory/hosts.example

```diff
@@ -147,10 +147,8 @@ debug_level=2
 # The following options must not be used
 # - openshift_docker_options
 #openshift_docker_use_system_container=False
-# Install and run cri-o. By default this will install cri-o as a system container.
+# Install and run cri-o.
 #openshift_use_crio=False
-# You can install cri-o as an rpm by setting the following variable:
-#openshift_crio_use_rpm=False
```

Comment:
Should we add `openshift_crio_use_rpm` to `__warn_deprecated_vars`?

Reply:
Yeah, we should do that.  Make a note to just remove in 3.12, no need to fail if that's defined as it won't really affect anything.